src/actors/builtin/init/init_actor.go

```diff
@@ -87,15 +96,27 @@ func (s *InitActorState_I) ResolveAddress(address addr.Address) addr.Address {
 	return address
 }
 
-func (s *InitActorState_I) MapAddressToNewID(address addr.Address) addr.Address {
-	actorID := s.NextID_
-	s.NextID_++
-	s.AddressMap()[address] = actorID
+func (s *InitActorState) MapAddressToNewID(address addr.Address) addr.Address {
+	actorID := s.NextID
+	s.NextID++
+	s.AddressMap[address] = actorID
 	idAddr, err := addr.NewIDAddress(uint64(actorID))
 	autil.Assert(err == nil)
 	return idAddr
 }
 
+func (st *InitActorState) CID() cid.Cid {
+	panic("TODO")
+}
+
+func Deserialize_InitActorState(Bytes) (InitActorState, error) {
+	panic("TODO")
+}
+
+func Deserialize_InitActorState_Assert(Bytes) InitActorState {
+	panic("TODO")
+}
+
```

Comment:
required by `vm_interpreter.go`

Reply:
Replace with use of the actors/serde module.